### ISSUE TITLE
Samsung Display Control

### ISSUE THREAD

**Issue by scienceworld** [museumsvictoria/nodel-recipes#5]
Can you give a little insight into how the samsung control recipe works?
It appears to be via IP.  Is there any special configuration to the display that needs to happen to support this, or any specific model this works with?

We have MD55-C and DM55-E models of samsung displays, and wonder if we can get this working with them.

**Comment by scienceworld** [comment]
Is this using MDC (Multiple Display Control) protocol that works over rs232 and tcp/ethernet?

**Comment by scienceworld** [comment]
got it to work on a samsung DM55E display.  had to enable network standby whereby the network is active when the device is in standby.  This model doesnt have explicit MDC settings, so the panel ID is just 0.  it worked by default.

Was trying to do this on an MD55C which does have explicit MDC settings.  I set it to RJ45 ethernet and panel id 0.  I was able to turn the display off.  However when the display does turn off it drops its IP address but keeps ethernet link state up.  Very odd.  I couldnt find network standby settings on this guy.
